.github/fabricbot.json

```diff
@@ -1,965 +0,0 @@
-            }
-          }
-        ]
-      },
-      "id": "0jjnItGjnF"
-    },
-    {
-      "taskType": "scheduled",
-      "capabilityId": "ScheduledSearch",
-      "subCapability": "ScheduledSearch",
-      "version": "1.1",
-      "config": {
-        "frequency": [
-          {
-            "weekDay": 0,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 1,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 2,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 3,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 4,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 5,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 6,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          }
-        ],
-        "searchTerms": [
-          {
-            "name": "isClosed",
-            "parameters": {}
-          },
-          {
-            "name": "noActivitySince",
-            "parameters": {
-              "days": 30
-            }
-          },
-          {
-            "name": "isUnlocked",
-            "parameters": {}
-          },
-          {
-            "name": "isIssue",
-            "parameters": {}
-          }
-        ],
-        "taskName": "Lock issues closed without activity for over 30 days",
```

Comment:
@mkArtakMSFT @wtgodbe This scheduled search did not get ported because it is operating on closed issues. We need to adopt Locker to replace this task.